test/common-tests/environment-variable-parsing-unit-tests.ts

```diff
@@ -30,18 +37,79 @@ export const withEnvironmentVariableParsingUnitTests = <ConfigUnderTest>({
     process.env = originalProcessEnv;
   });
 
-  describe.each(configDirectlyFromEnvironmentVariables)('$configPropertyName', ({ configPropertyName, environmentVariableName }) => {
-    it(`is the env variable ${environmentVariableName}`, () => {
-      const environmentVariableValue = valueGenerator.string();
-      process.env = {
-        [environmentVariableName]: environmentVariableValue,
-      };
+  if (configDirectlyFromEnvironmentVariables) {
+    describe.each(configDirectlyFromEnvironmentVariables)('$configPropertyName', ({ configPropertyName, environmentVariableName }) => {
+      it(`is the env variable ${environmentVariableName}`, () => {
+        const environmentVariableValue = valueGenerator.string();
+        process.env = {
+          [environmentVariableName]: environmentVariableValue,
+        };
 
-      const { [configPropertyName]: configPropertyValue } = getConfig();
+        const { [configPropertyName]: configPropertyValue } = getConfig();
 
-      expect(configPropertyValue).toBe(environmentVariableValue);
+        expect(configPropertyValue).toBe(environmentVariableValue);
+      });
     });
-  });
+  }
+
+  if (configParsedBooleanFromEnvironmentVariablesWithDefault) {
+    describe.each(configParsedBooleanFromEnvironmentVariablesWithDefault)(
+      '$configPropertyName',
+      ({ configPropertyName, environmentVariableName, defaultConfigValue }) => {
+        it(`is config ${environmentVariableName} set to true if environment variable ${environmentVariableName} is true`, () => {
```

Comment:
I think `is true if environment variable ${environmentVariableName} is true` reads a bit better here